examples/java/src/test/java/org/apache/beam/examples/WordCountIT.java

```diff
@@ -55,10 +55,5 @@ public void testE2EWordCount() throws Exception {
         options.getJobName(), "output", "results"}));
 
     WordCount.main(TestPipeline.convertToArgs(options));
```

Comment:
I like the explicit nature of the verifier here. Perhaps we can keep that -- not have a "default" verifier.